lib/dissoc/dissoc.performance.js

```diff
@@ -0,0 +1,17 @@
+const generateObjects = require('../_internal/helpers/performance').generateObjects
```

Comment:
maybe `const { generateObjects } = require('../_internal/helpers/performance')`? It's shorter

Reply:
Yeah, agree